dart/lib/src/sentry_traces_sampler.dart

```diff
@@ -50,11 +52,16 @@ class SentryTracesSampler {
       return parentSamplingDecision;
     }
 
-    final tracesSampleRate = _options.tracesSampleRate;
-    if (tracesSampleRate != null) {
+    double? optionsRate = _options.tracesSampleRate;
+    double? defaultRate = _options.enableTracing == true
+        ? SentryTracesSampler._defaultSampleRate
```

Comment:
```suggestion
        ? _defaultSampleRate
```

you probably dont need the class name?